scripts/download_file.py

```diff
@@ -141,11 +148,10 @@ def download_file(url: str, dest_file: str) -> bool:
         return False
 
     with open(dest_file, "wb") as f:
-        while True:
-            buffer = u.read(BLOCK_SIZE)
-            if not buffer:
-                break
-            f.write(buffer)
+        chunk = u.read(BLOCK_SIZE)
+        while chunk is not None:
+            f.write(chunk)
+            chunk = u.read(BLOCK_SIZE)
```

Comment:
This code is not working as expected for me. The While loop never ends. Makes me suspect that the implicit comparison was not detecting an explicit None but instead checking for a var value that is equivalent to False (an empty string, 0, etc).

Reply:
Addressed.